paddle/fluid/framework/executor.cc

```diff
@@ -68,7 +63,11 @@ ExecutorPrepareContext::~ExecutorPrepareContext() {
   VLOG(5) << "destroy ExecutorPrepareContext";
 }
 
-Executor::Executor(const platform::Place& place) : place_(place) {}
+Executor::Executor(const platform::Place& place) : place_(place) {
+  cur_step_ = 0;
```

Comment:
what if i use this exe to run other programs, like startup program. Do we need a reseting method for context pamameters?